python/demo/pyvista/demo_pyvista.py

```diff
@@ -176,8 +173,8 @@ def in_circle(x):
 # Create a dolfinx.MeshTag for all cells. If midpoint is inside the
 # circle, it gets value 1, otherwise 0.
 num_cells = mesh.topology.index_map(mesh.topology.dim).size_local
-midpoints = dolfinx.mesh.midpoints(mesh, mesh.topology.dim, list(np.arange(num_cells, dtype=np.int32)))
-cell_tags = dolfinx.MeshTags(mesh, mesh.topology.dim, np.arange(num_cells), in_circle(midpoints))
+midpoints = midpoints(mesh, mesh.topology.dim, list(np.arange(num_cells, dtype=np.int32)))
```

Comment:
This overwrites the `dolfinx.mesh.midpoints` placeholder.

Reply:
Good point. I wanted to change the function name because it should be a verb.

Reply:
`mesh.midpoints` renamed to `mesh.compute_midpoints`